src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -150,10 +150,14 @@ public async Task<int> DeployAsync()
                 azureSubscriptionClient = azureClient.WithSubscription(configuration.SubscriptionId);
                 subscriptionIds = (await azureClient.Subscriptions.ListAsync()).Select(s => s.SubscriptionId);
                 resourceManagerClient = GetResourceManagerClient(azureCredentials);
-                kubernetesManager = new KubernetesManager(configuration, azureCredentials, cts);
                 networkManagementClient = new Microsoft.Azure.Management.Network.NetworkManagementClient(azureCredentials) { SubscriptionId = configuration.SubscriptionId };
-                postgreSqlFlexManagementClient = new FlexibleServer.PostgreSQLManagementClient(azureCredentials) { SubscriptionId = configuration.SubscriptionId };
-                postgreSqlSingleManagementClient = new SingleServer.PostgreSQLManagementClient(azureCredentials) { SubscriptionId = configuration.SubscriptionId };
+                postgreSqlFlexManagementClient = new FlexibleServer.PostgreSQLManagementClient(azureCredentials) { SubscriptionId = configuration.SubscriptionId, LongRunningOperationRetryTimeout = 1200 };
+                postgreSqlSingleManagementClient = new SingleServer.PostgreSQLManagementClient(azureCredentials) { SubscriptionId = configuration.SubscriptionId, LongRunningOperationRetryTimeout = 1200 };
+
+                if (configuration.UseAks)
```

Comment:
Because of [this line](https://github.com/microsoft/CromwellOnAzure/blob/45e06037a1d32b7d4bcdf565839e01f4944c16f1/src/deploy-cromwell-on-azure/Deployer.cs#L192), there may be a NullReferenceException during Updates. I suggest you move this block down (twice, once in update path, other in initial deployment path) to closer to where it's first needed

Reply:
Thank you, there is no negative impact keeping it where it is, so am going to leave it.  I put it in two different places but then seems a bit counterintuitive why it's there and instantiated in two different places.